lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'menu'
+#require_relative 'make_call'
+
+
+class Order
+
+attr_reader :order, :phone_number, :order, :cust_order
+
+    def initialize
+        @menu = Menu.new
+        @order = nil
+        @phone_number = nil
+        @cust_order = []
+      #@checkout = Checkout.new
+    end
+
+    def view_menu
+        puts "Todays menu is as follows..."
+        @menu.todays_menu
+        puts "#{@menu.print_menu}"
+    end
+
+    def place_order
+          puts "... please select your pizza and press enter"
```

Comment:
User interface logic should not be in this class - hopefully now you've been introduced to Separation of Concerns, you can see why.  The `Order` class should only be responsible for representing the selected dishes and their quantities (and probably the total).